ejemplo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The Cheese Guardian</title>
+  <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <section class="main-container">
+    <img src="phone.png" alt="foto"  />
```

Comment:
alt tags are used to display something when the image doesn't load, you should use a more descriptive text (in english please).